src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -344,9 +344,9 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_expm1)
 
 // cos
 MXNET_OPERATOR_REGISTER_UNARY(cos)
-.describe(R"code(Cosine, element-wise.
+.describe(R"code(Returns element-wise trigonometric cosine value of the input.
 
-The input is in radians (:math:`2\pi` rad equals 360 degress).
+The input is in radians (:math:`2\pi` rad equals 360 degrees).
```

Comment:
Better to say:

"The input should be specified in radians (:math:`2\pi` rad equals 360 degrees)."